File: app/src/main/java/im/point/dotty/network/PostsReply.java
package im.point.dotty.network;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public final class PostsReply extends Envelope {
    @SerializedName("has next") boolean hasNext;
    List<MetaPost> posts;

    public List<MetaPost> getPosts() {
        return posts;
    }
}

File: app/src/main/java/im/point/dotty/network/PostsReply.kt
/*
 * Copyright (c) 2019-2021 Pavel A. Sokolov
 */
package im.point.dotty.network

import com.google.gson.annotations.SerializedName

data class PostsReply(@SerializedName("has_next")
                      var hasNext: Boolean = false,
                      var posts: List<MetaPost> = listOf()) : Envelope()